bin/pycbc_multi_inspiral

```diff
@@ -197,18 +195,61 @@ def get_weighted_antenna_patterns(Fp_dict, Fc_dict, sigma_dict):
     return wp, wc
 
 
-def get_projection_matrix(wp, wc):
+def get_projection_matrix(wp, wc, projection_type='standard'):
     """
     Output: projection_matrix: Projects the data onto the signal space
     Input:  wp,wc: The weighted antenna response fuctions to plus and cross
                    polarisations respectively
+            projection_type: standard, left or right
+    """
+    standard_denominator = np.dot(wp, wp) * np.dot(wc, wc) - np.dot(wp, wc)**2
+    standard_projection_matrix = (np.dot(wc, wc)*np.outer(wp, wp) +
+                                  np.dot(wp, wp)*np.outer(wc, wc) -
+                                  np.dot(wp, wc)*(np.outer(wp, wc) +
+                                  np.outer(wc, wp))) / standard_denominator
+
+    side_denominator = np.dot(wp, wp) + np.dot(wc, wc)
+    left_projection_matrix = (np.outer(wp, wp) + np.outer(wc, wc) +
+                              (np.outer(wp, wc) - np.outer(wc, wp))
+                              *1j) / side_denominator
+    right_projection_matrix = (np.outer(wp, wp) + np.outer(wc, wc) +
+                               (np.outer(wc, wp) - np.outer(wp, wc))
+                               *1j) / side_denominator
+ 
+    if projection_type=='standard':
+        logging.info("choice of projection matrix is %s"
+                            % str(projection_type))
```

Comment:
Calls to `logging.info()` should not use the string formatting operations (`%` and `.format()`). Instead, do this:
```python
logging.info("choice of projection matrix is %s", projection_type)
```
(this comment also applies to `logging.info()` further below).